scripts/extract_class_content.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python3
+"""Script to convert linkML schema to JSON schema and extract the class contents into
+individual files"""
+
+import json
+import os
+import subprocess
+from pathlib import Path
+from typing import Union
+
+import yaml
+from pydantic import BaseModel, TypeAdapter
+from script_utils.cli import run
+
+ROOT = Path(__file__).parent.parent.resolve()
+LINKML_SCHEMA_PATH = ROOT / "src" / "schema" / "submission.yaml"
+CLASS_CONTENT_DIR = ROOT / "src" / "content_schemas"
+ENUM_CLASS_CONTENT_DIR = CLASS_CONTENT_DIR / "enums"
+RELATIONS_CONFIG = ROOT / "relations_config.yaml"
+EXCLUDE_CONFIG = ROOT / "exclude_config.yaml"
+
+
+class Relation(BaseModel):
+    """Model to represent the relation of a class."""
+
+    name: str
+    relations: list[Union[None, str]]
+
+
+def load_yaml(path: Path):
+    """Load the yaml file content."""
+    with path.open("r", encoding="utf-8") as file:
+        data = yaml.safe_load(file)
+    return data
+
+
+def _reshaped_relation(path: Path = RELATIONS_CONFIG):
+    """Reshape relations, convert it from a nested structure to a simple
+    list with relation names."""
+    data = load_yaml(path)
+    return [
+        {"name": key, "relations": value["relations"].keys()}
+        for key, value in data["classes"].items()
+    ]
+
+
+def load_config():
+    """Load the relations from relations_config.yaml."""
+
+    def _construct_config(relations_config):
+        return TypeAdapter(list[Relation]).validate_python(relations_config)
+
+    relations_config = _reshaped_relation()
+    return _construct_config(relations_config)
+
+
+def linkml_to_json(file: Path) -> dict:
+    """Create a JSON representation of the schema described in LinkML using LinkML
+    command line option."""
+    with subprocess.Popen(["gen-json-schema", file], stdout=subprocess.PIPE) as process:
+        stdout, _ = process.communicate()
+    return json.loads(stdout.decode("utf-8"))
```

Comment:
You may want to add some error handling here for `subprocess.CalledProcessError` when command fails and `json.JSONDecodeError` when json decoding fails

Reply:
Thanks for the suggestion, but it's okay to leave it as is. This script won't be part of the repository—it's just a helper to convert the LinkML to SchemaPack, and it won't be needed once we complete the transition. Also, the script wouldn't work with the current `requirements.txt` file anyway since we've already removed all the LinkML-related packages. It is a part of the PR for archiving purposes :)